docs/building-a-node-with-ldk/introduction.md

```diff
@@ -0,0 +1,36 @@
+# Building a Node with LDK
+
+## Learn how to build a basic LDK node from scratch
+
+::: tip Note
+For an integrated example of an LDK node in Rust, see the [Sample Node](https://github.com/lightningdevkit/ldk-sample)
+:::
+
+The following tutorials will show you how to build the simplest lightning node using LDK, that fufills the following tasks:
+
+1. **Connecting to Peers**
+2. **Opening Channels**
+3. **Sending Payments**
+4. **Receiving Payments**
+5. **Closing Channels**
+
+### Foundational Components
+
+Let's start by looking at the core components we'll need to make this node work for the tasks we outlined above.
+
+1. A [`PeerManager`](https://docs.rs/lightning/*/lightning/ln/peer_handler/struct.PeerManager.html), for establishing TCP/IP connections to other nodes on the lightning network.
+2. A [`ChannelManager`](https://docs.rs/lightning/*/lightning/ln/channelmanager/struct.ChannelManager.html), to open and close channels.
```

Comment:
May want to switch the order of these two given the order they are covered.

Reply:
Fixed in https://github.com/lightningdevkit/lightningdevkit.org/pull/156/commits/31f155f70dbda0759c1bc17c57e61423ce182892